src/modules/common/providers/UserContext.js

```diff
@@ -11,9 +12,36 @@ export const UserProvider = ({ children }) => {
         email: '',
         token: ''
     })
+    useEffect(() => {
+        async function validateToken() {
+            const token = localStorage.getItem('userToken')
+            const response = await axios.get(
```

Comment:
can we wrap this whole thing in an if
```
if (token) { ...
```

That way if its first time load we dont make the call with a. null token

Reply:
also wrap it all in a try catch yeah

Reply:
done